src/app/content/components/Toolbar/PracticeQuestionsButton.tsx

```diff
@@ -58,17 +49,15 @@ const PracticeQuestionsButton = () => {
 
   if (!isEnabled || !hasPracticeQs || !book || !page) { return null; }
 
-  const openPracticeQuestions = () => {
-    dispatch(replace(match as AnyMatch, {
-      search: getQueryForParam({[modalQueryParameterName]: modalUrlName}, existingQuery),
-    }));
+  const showPracticeQuestions = () => {
+    dispatch(openPracticeQuestions());
     trackOpenClose();
   };
 
   const text = intl.formatMessage({id: 'i18n:toolbar:practice-questions:button:text'});
 
   return <StyledPracticeQuestionsButton
-    onClick={() => openPracticeQuestions()}
+    onClick={() => showPracticeQuestions()}
```

Comment:
Since it doesn't care about args, it could just be `onClick={showPracticeQuestions}`, right?